src/Collection/Modifier/InsertValueBeforeKey.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Arrays\Collection\Modifier;
+
+use Yiisoft\Arrays\Collection\Modifier\ModifierInterface\DataModifierInterface;
+
+/**
+ * Modifier "Insert Value Before Key".
+ *
+ * Перемещает элемент с ключом "key" на позицию перед элементом с ключом "beforeKey".
```

Comment:
That contradicts the name of the modifier. It doesn't insert, it move from key a to key b.

Reply:
Renamed to `MoveValueBeforeKey`.